sqlitecluster/SQLiteNode.cpp

```diff
@@ -56,6 +56,20 @@
 // dbCountAtStart:   The highest committed transaction in the DB at the start of this transaction on leader, for
 //                   optimizing replication.
 
+// NOTE: This comment as well as NODE_LOGIN should be removed after https://github.com/Expensify/Bedrock/pull/1999 is deployed.
```

Comment:
Is this tracked in the same issue?

Reply:
https://github.com/Expensify/Expensify/issues/450953